translate/tracer.go

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018 De-labtory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package translate
+
+// Identifier saves size and offset of the value which the variable has.
+type Identifier struct {
+	Offset uint
+	Size   uint
+}
+
+// MemoryTable is used to know the location of the memory
+type MemoryTable struct {
```

Comment:
~~I think in `MemoryTable`, we would save both identifier (variable name) and its value.~~ I think `MemoryTableEntry` would be better than `Identifier`.

ref:
* page table: https://www.geeksforgeeks.org/operating-system-page-table-entries/
* page table entry: https://www.geeksforgeeks.org/operating-system-page-table-entries/

I know that page table is not exactly same as memory table, but I posted the link because we may can use that words for similar concepts

Reply:
Thanks!